src/index.html

```diff
@@ -7,8 +7,30 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,500;1,500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#home">
+        <img class="logo"
+        src="images/logo.png"
+        alt="#">
```

Comment:
```suggestion
        <img 
          class="logo"
          src="images/logo.png"
          alt="#"
        >
```
![image](https://github.com/mate-academy/layout_moyo-header/assets/96409563/7f87733e-cd5a-4c44-9664-6954f937666b)

Reply:
+